prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -433,6 +433,22 @@ presubmits:
         - prow/e2e-simpleTests.sh
       nodeSelector:
         testing: test-pool
+  - name: e2e-simpleTests-distroless-master
+    <<: *job_template
+    optional: true
+    context: prow/e2e-simpleTests-distroless.sh
+    labels:
+      preset-service-account: "true"
+    max_concurrency: 5
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - scripts/build_test_presubmit.sh
```

Comment:
Just based on it not being in the other ones, I am a bit suspicious of this. @icygalz is the expert on this so I'll let her take a look, otherwise lgtm

Reply:
Why do you want to run this script?
https://github.com/istio-releases/pipeline/blob/master/scripts/build_test_presubmit.sh

Reply:
@utka, the distroless docker images are build in `prow/istio-presubmit.sh` called from `build_test_presubmit.sh`. These images are required for this test.